src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -157,7 +157,10 @@ public bool IsOptional
         /// </summary>
         internal abstract bool IsMetadataOptional { get; }
 
-        internal abstract bool IsNullChecked { get; }
+        /// <summary>
+        /// True if parameter will throw when null.
```

Comment:
>True if parameter will throw when null. [](start = 12, length = 39)

I would say this should be `True if the containing member will throw if this parameter is null`.

Reply:
As the parameter does not throw on its own.

---
In reply to: [308373581](https://github.com/dotnet/roslyn/pull/37381#discussion_r308373581) [](ancestors = 308373581)

Reply:
The comment should mention that the null check is generated by the compiler, in contrast to an argument that is explicitly checked in source.

Perhaps: "True if the compiler will synthesize a null check for this parameter (the parameter is declared in source with a '!' following the parameter name)."

---
In reply to: [308373665](https://github.com/dotnet/roslyn/pull/37381#discussion_r308373665) [](ancestors = 308373665,308373581)